docs/markdown/Reference-manual.md

```diff
@@ -689,18 +689,18 @@ The `meson` object allows you to introspect various properties of the system. Th
 
 ### `build_machine` object
 
-Provides information about the build machine — the machine that is doing the actual compilation. See [Cross-compilation](Cross-compilation.md). It has the following methods:
+Provides information about the build machine — the machine that is doing the actual compilation. See [Cross-compilation](Cross-compilation.html). It has the following methods:
 
 - `cpu_family()` returns the CPU family name. Guaranteed to return `x86` for 32-bit userland on x86 CPUs, `x86_64` for 64-bit userland on x86 CPUs, `arm` for 32-bit userland on all ARM CPUs, etc.
 - `cpu()` returns a more specific CPU name, such as `i686`, `amd64`, etc.
 - `system()` returns the operating system name, such as `windows` (all versions of Windows), `linux` (all Linux distros), `darwin` (all versions of OS X), etc.
 - `endian()` returns `big` on big-endian systems and `little` on little-endian systems.
 
-Currently, these values are populated using the [`platform.system()`](https://docs.python.org/3.4/library/platform.html#platform.system) and [`platform.machine()`](https://docs.python.org/3.4/library/platform.html#platform.machine). If you think the returned values for any of these are incorrect for your system or CPU, please file [a bug report](/mesonbuild/meson/issues/new).
+Currently, these values are populated using the [`platform.system()`](https://docs.python.org/3.4/library/platform.html#platform.system) and [`platform.machine()`](https://docs.python.org/3.4/library/platform.html#platform.machine). If you think the returned values for any of these are incorrect for your system or CPU, please file [a bug report](https://github.com/mesonbuild/meson/issues/new).
 
 ### `host_machine` object
 
-Provides information about the host machine — the machine on which the compiled binary will run. See [Cross-compilation](Cross-compilation.md).
```

Comment:
Yes, i've reproduced the issue, happens in incremental mode, looking into it and I'll release a fix later today along with a few other things, please don't merge that specific patch it's indeed incorrect